src/components/SettingsPage.vue

```diff
@@ -255,7 +255,22 @@ import AdSettings from "@/components/adset.vue";
 export default {
   name: "SettingsPage",
   components: { AdSettings },
-};
+  data () {
+    return {
+      username: '',
+      submitted: false
+    }
+},
+methods: {
```

Comment:
`methods` is not being closed, please see to this. This is why error is being raised.